examples/seismic/tutorials/06_elastic.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>bis?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2367/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
It's a latin word meaning "twice", used for adresses/musci/...  when need to insert a duplicate number 

https://www.oxfordreference.com/display/10.1093/oi/authority.20110803095508312